components/apimgt/org.wso2.carbon.apimgt.rest.api.util/src/main/java/org/wso2/carbon/apimgt/rest/api/util/RestApiConstants.java

```diff
@@ -206,6 +206,9 @@
     public static final String SORT_BY_THROTTLING_TIER = "throttlingTier";
     public static final String SORT_BY_STATUS = "status";
 
+    //Constants for endpoint registry sortBy
+    public static final String SORT_BY_ENDPOINT_REG_NAME = "registryName";
```

Comment:
let's name it generically. ENDPOINT_REG_NAME

Reply:
we can add this as well to EndpointRegistryConstants.java?

Reply:
We can. I added it here because it is a REST API level parameter name

Reply:
ok, let's keep it here.

Reply:
updated the constant name